docker/docker-jaxrs/pom.xml

```diff
@@ -9,25 +9,25 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <artifactId>examples-docker</artifactId>
-    <groupId>org.wildfly.swarm.examples</groupId>
-    <version>2018.6.0-SNAPSHOT</version>
+    <groupId>io.thorntail.examples</groupId>
+    <version>2.0.0.Final-SNAPSHOT</version>
     <relativePath>../</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>example-docker-jaxrs</artifactId>
 
-  <name>WildFly Swarm Examples: Docker and JAX-RS</name>
-  <description>WildFly Swarm Examples: JAX-RS in Docker container</description>
+  <name>Thorntail Examples: Docker and JAX-RS</name>
```

Comment:
`-swarm.jar` is used in `docker-maven-plugin` configuration below. Note that `fabric8-maven-plugin` needs upgrade too - see also https://issues.jboss.org/browse/THORN-2048.

Reply:
Is there a temporary release of the plugin or do we have to wait for the release?